File: StudentSCore.java
import java.util.Scanner;
	public class StudentSCore {
		public static void main(String[] args) {
		
	Scanner input = new Scanner(System.in);

		
		
		int studentFailed = 0;
		int studentPassed = 0;
		
		for(int counter = 1; counter <= 15; counter++) {
			System.out.print("Enter a score for student");
			int userInput = input.nextInt();	


			if(userInput < 45) {
			studentFailed++;
			}
			
			if(userInput >= 45) {
			studentPassed++;
			}
		}
			System.out.println("These are the students that failed " +studentFailed);
			System.out.print("These are the students that passed " +studentPassed);
	}
}

File: Chapter 3 assignment/student_score.py
passes = 0
failures = 0
#user_input = int(input("Enter result (1 = pass, 2 = fail): "))

while True:
	user_input = int(input("Enter result (1 = pass, 2 = fail): "))

	if user_input == 1:
		passes = passes + 1
	if user_input == 2:
		failures = failures + 1	
	
	if user_input > 1 and user_input > 2:
		break
	

print('Passed: ', passes)
print('Failed: ', failures)

if passes > 8:
	print('Bonus to instructor')